kivy/uix/textinput.py

```diff
@@ -1562,13 +1565,61 @@ def on_touch_move(self, touch):
             if self._selection_touch is touch:
                 self._selection_touch = None
             return False
-        if self._selection_touch is touch:
+
+        # multiline = True
+        if self.scroll_from_swipe:
+            if self.multiline:
+                _scroll_timeout = touch.time_update - touch.time_start
+                self._scroll_distance_x += abs(touch.dx)
+                self._scroll_distance_y += abs(touch.dy)
+
+                # disable scroll and start selection mode if scroll distance
+                # isn't reached within scroll_timeout
+                if _scroll_timeout >= self.scroll_timeout/1000 and\
+                   self._scroll_distance_x <= self.scroll_distance and\
+                   self._scroll_distance_y <= self.scroll_distance:
```

Comment:
this indent is not obvious enough, one way to make it more apparent is to parenthesize the expression like this
```suggestion
                if (
                    _scroll_timeout >= self.scroll_timeout/1000
                    and self._scroll_distance_x <= self.scroll_distance
                    and self._scroll_distance_y <= self.scroll_distance
                ):
```